docs/features/details/Details.md

```diff
@@ -29,47 +27,58 @@ An object's further detail properties as entries with
                  e.g. a simple string or a more complex object
 end note
 
+Branch --> Details : details
```

Comment:
Add multiplicity (1) to associations of _Branch, Build, UseCase, Scenario, Step_

Reply:
I find this not so important - If I not use any multiplicity it means always (1) - this just clutters the UML diagrams unnecessarily IMHO.

Reply:
as discussed I will try to add it none the less

Reply:
done